packages/iTwinUI-react/src/core/Wizard/Wizard.tsx

```diff
@@ -67,61 +65,28 @@ const defaultWizardLocalization: WizardLocalization = {
  *  type='long'
  *  />
  */
-export const Wizard = React.forwardRef<HTMLDivElement, WizardProps>(
-  (props, ref) => {
-    const {
-      currentStep,
-      steps,
-      type = 'default',
-      localization = defaultWizardLocalization,
-      onStepClick,
-      ...rest
-    } = props;
+export const Wizard = (props: WizardProps) => {
+  const {
+    currentStep,
+    steps,
+    type = 'default',
+    localization = defaultWizardLocalization,
```

Comment:
Is the definition for `defaultWizardLocalization` (line 50) needed anymore if the localization default is defined in `Stepper`?

Reply:
`defaultWizardLocalization` removed for `Wizard`